src/sync.rs

```diff
@@ -127,3 +132,10 @@ fn summary_message(installed_tools: u64, store_directory: PathBuf) {
         store_directory.display()
     );
 }
+
+fn empty_prefetched_tool_assets_message() {
+    eprintln!(
+        r"No tools are required to sync: configuration is empty 
+or encountered error prefetching tools."
```

Comment:
We check if the configuration is empty earlier. So here it can be empty only if all tools have errors.

I suggest changing the `configuration is empty` to `nothing to sync` in case we implement #70 in the future (just to future-proof the message a bit). In that case, the list also will be empty if all versions are up-to-date.

```suggestion
        r"No tools are required to sync: encountered error prefetching tools or nothing to sync."
```